protobuf/types/process.proto

```diff
@@ -69,25 +86,43 @@ message Process {
           ];
         }
 
-        // Key of the output.
-        string key = 1 [
-          (gogoproto.moretags) = 'hash:"name:1" validate:"required"'
-        ];
-
         oneof value {
+          Null null = 1 [
+            (gogoproto.moretags) = 'hash:"name:1" validate:"required_without=Ref Constant List Map"'
+          ];
+
+          string string_const = 2 [
+            (gogoproto.moretags) = 'hash:"name:2"'
+          ];
+
+          double double_const = 3 [
+            (gogoproto.moretags) = 'hash:"name:3"'
+          ];
+
+          bool bool_const = 4 [
+            (gogoproto.moretags) = 'hash:"name:4"'
+          ];
+
           // Input defined as reference.
-          Reference ref = 2 [
-            (gogoproto.moretags) = 'hash:"name:2" validate:"required_without=Constant"'
+          Reference ref = 5 [
+            (gogoproto.moretags) = 'hash:"name:5" validate:"required_without=Null Constant List Map"'
```

Comment:
Is it working and necessary: `required_without=Null Constant List Map`? it's already in a oneof.

Reply:
Add the key name are wrong..

Reply:
If we receive the data from grpc that shouldn't be necessary but if we use directly the struct then this will trigger and error when we validate the struct.

Reply:
@krhubert can you fix the validation here, we need to add the string/double/bool